javascript/packages/orchestrator/src/providers/k8s/chain-spec.ts

```diff
@@ -8,7 +8,7 @@ import { getClient } from "../client";
 import { createTempNodeDef, genNodeDef } from "./dynResourceDefinition";
 const debug = require("debug")("zombie::kube::chain-spec");
 
-const fs = require("fs").promises;
+const { writeFileSync, promises } = require("fs");
```

Comment:
Can we rename `promises`? e.g somethig less generic.
```suggestion
const { writeFileSync, promises as fsPromises } = require("fs");
```